repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -226,7 +226,13 @@ class Interpreter(prompt0: Ref[String],
         eval.addJar(jar.toURI.toURL)
       }
 
-      def apply(line: String) = processExec(line)
+      def apply(line: String, visible: Boolean) = {
+        if (!visible) processExec(line)
```

Comment:
Could we make `processExec` take an optional `stdout` argument, instead of duplicating the code below?